Buffer.h

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
 #ifndef BUFFER_SIZE
-#define BUFFER_SIZE 200
+#define BUFFER_SIZE 100
```

Comment:
This number controls how many values the buffer stores. Apogee is detected when half of the buffer is descending. At 200Hz, 200 samples is 1 second, so on master it triggers half a second after actual apogee. Changing this to 100 means it triggers a quarter-second after apogee.